.github/workflows/embedded.yml

```
@@ -52,6 +52,21 @@ jobs:
           RUNTIME_TEST_DISABLE: builtin.cgroup,probe.kprobe_offset_fail_size
           BASE: bionic
           DISTRO: ubuntu-glibc
+        - TYPE: Release
+          NAME: vanilla_llvm12+clang+glibc2.27
+          LLVM_VERSION: 12
+          STATIC_LINKING: ON
+          STATIC_LIBC: OFF
+          EMBED_LLVM: ON
+          EMBED_CLANG: ON
+          EMBED_LIBCLANG_ONLY: OFF
+          EMBED_BCC: OFF
+          EMBED_LIBELF: OFF
+          EMBED_BINUTILS: OFF
+          RUN_ALL_TESTS: 1
+          RUNTIME_TEST_DISABLE: builtin.cgroup,probe.kprobe_offset_fail_size
```

Didn't your other PR remove `builtin.cgroup`?
```suggestion
          RUNTIME_TEST_DISABLE: probe.kprobe_offset_fail_size
```

---

yes, looks like i forgot to update this flie.